msteams-platform/concepts/device-capabilities/people-picker-capability.md

```diff
@@ -65,6 +65,17 @@ The People Picker control on web or desktop is launched in a modal window on top
 
 The following code snippet displays use of the `selectPeople` API people from a list:
 
+# [TeamsJS v2](#tab/teamsjs-v2)
+
+```javascript
+microsoftTeams.people.selectPeople({ setSelected: ["aad id"], openOrgWideSearchInChatOrChannel: true, singleSelect: false}).then(people) => 
```

Comment:
microsoftTeams prefix should be removed

Reply:
also, I believe "title" is a property you can set on the selectPeople call, not sure if that's an intentional omission or not but probably worth also specifying it here for completeness